src/components/ListingDetail/ListingDetail.js

```diff
@@ -0,0 +1,38 @@
+import { getListing } from '../../api/glue-api'
+import { mapGetters } from 'vuex'
+
+export default ({
+  name: 'ListingDetail',
+
+  data() {
+    return {
+      isLoading: false,
+      listing: {}
+    }
+  },
+  
+  computed: {
+    ...mapGetters(['getHrefFromId']),
+  },
+
+  methods: {
+    getIdByPathname() {
+      const id = window.location.pathname
+      return id.replace('/listing/', '')
+    },
+
+    listingHref() {
+      const id = this.getIdByPathname()
+      const href = this.getHrefFromId(id)
+
+      return href
+    }
```

Comment:
You don't need of  const href. Just return directly.

```suggestion
    listingHref() {
      const id = this.getIdByPathname()
      return this.getHrefFromId(id)
    }
```

Reply:
True, true.